Dockerfile

```diff
@@ -1,11 +1,19 @@
+# Golang version
 FROM golang:1.8
 
 MAINTAINER rafaelrendonpablo@gmail.com
 
+# Get the dependencies
 RUN go get github.com/onsi/ginkgo/ginkgo &&  go get github.com/onsi/gomega
 
+# Get & Build API
+RUN go get github.com/algoristas/api
+RUN env GOOS=linux GOARCH=amd64 go build -o bin/api github.com/algoristas/api
+
+# Create application folder
 RUN mkdir -p /app
 
+# Set working directory
```

Comment:
I'm not sure, but seems like I forgot to move the the working directory (`cd /app`), could you verify please?